core/tests/org.fusesource.ide.camel.model.service.core.tests.integration/META-INF/MANIFEST.MF

```diff
@@ -16,9 +16,10 @@ Require-Bundle: org.eclipse.core.filesystem,
  org.jboss.tools.locus.mockito;bundle-version="1.9.5",
  org.fusesource.ide.foundation.core;bundle-version="10.0.0",
  org.fusesource.ide.foundation.ui;bundle-version="10.0.0",
- org.fusesource.ide.camel.model.service.core,
- org.fusesource.ide.camel.model.service.impl,
+ org.fusesource.ide.camel.model.service.core;bundle-version="10.0.0",
```

Comment:
is there any reason for requiring a specific version ?

Reply:
The tests will not work with bundles 9.x